test/integration/scheduler/scoring/priorities_test.go

```diff
@@ -151,23 +151,27 @@ func TestNodeResourcesScoring(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	cpuBoundPod1, err := runPausePod(testCtx.ClientSet, st.MakePod().Namespace(testCtx.NS.Name).Name("cpubound1").Req(
-		map[v1.ResourceName]string{
-			v1.ResourceCPU:    "2",
-			v1.ResourceMemory: "4G",
-			resourceGPU:       "1",
-		},
-	).Obj())
+
+	cpuBoundResourceMap := map[v1.ResourceName]string{
```

Comment:
If we want to reuse this variable through the test cases, then better to add this in the head of this function. But I suggest to revert this change, little improvements, and we forgot L146-L149, code style is inconsistent, WDYT?

Reply:
Reverted the variable changes.